storage/storage_tests/country_info_getter_test.cpp

```diff
@@ -37,19 +39,39 @@ UNIT_TEST(CountryInfoGetter_GetByPoint_Smoke)
   auto const getter = CreateCountryInfoGetter();
 
   CountryInfo info;
+  Storage storage;
+  bool const isSingleMwm = version::IsSingleMwm(storage.GetCurrentDataVersion());
 
   // Minsk
   getter->GetRegionInfo(MercatorBounds::FromLatLon(53.9022651, 27.5618818), info);
-  TEST_EQUAL(info.m_name, "Belarus", ());
-  TEST_EQUAL(info.m_flag, "by", ());
+  if (isSingleMwm)
+  {
```

Comment:
This code has been rewritten according to @syershov advice:
string const correctName = isSingleMwm ? "Belarus_Minsk Region" : "Belarus";
TEST_EQUAL(info.m_name, correctName, ());

It's impossible simple remove them because of the structure of  TEST_EQUAL